Traditional themes - use applet handles from menta

charlesbos opened the pull request:
1) Remove handles from the systray applet and the show desktop applet in both Traditional themes. They don't really add anything and, in my opinion, make the panel look more messy. See the vertical lines just to the left of the bluetooth icon in the screenshot: 

    ![screenshot at 2017-03-26 17-29-58](https://cloud.githubusercontent.com/assets/9774166/24333330/ba05d784-124d-11e7-84df-7a7b13c93f14.png)

2) There is inconsistency in the menu colors used for TraditionalGreen and TraditionalOk. TraditionalOk GTK+ 2 and TraditionalGreen GTK+ 2 & 3 use the base color which is white whilst TraditionalOk GTK+ 3 uses the background color which is grey. To bring TraditionalOk in line with everything else I changed the menu color to the base color. Aside from consistency, I prefer the white menus to the grey ones - I think they stand out more.

raveit65 commented:
The handles marks the right click area and in other themes i replaced them with an image.

sc0w commented:
1) If we remove it, the user can't see differences between icon launchers and sys tray icons, and we need it to know where is it, to handle it (holding left click to move, or right click with more options)

2) I prefer the currently color

so, I disagree with this PR

charlesbos commented:
That's fair. Two points though.

1) With regards to the handles, could they not at least be reduced from 4 vertical lines down to 2? The 4 lines seem a bit excessive to me.

2) Agree to disagree on the colours but can we not have consistency? GTK+ 2 applications still use white for both TraditionalGreen and TraditionalOk and TraditionalGreen GTK+ 3 is also still white. Why is TraditionalOk GTK+ 3 the odd one out here? Is this intentional?

lukefromdc commented:
What about a very subtle, barely visible handle that's there if you look for it, gives a place to drag the applet from, but is not visually obvious? That's similar to what I did in my own theme except I left it a little more visible as I just didn't want it at my normal grey background color on a black panel.

charlesbos commented:
That's a good idea, thanks! I had a look a what was done in BlackMate and Menta and I'd be happy if the traditional themes used those handles personally. I've updated my PR in that regard and removed the menu colour changes.

sc0w commented:
I have tested your new proposal, and I see nothing:

![pr169](https://cloud.githubusercontent.com/assets/7734191/24495805/9c54dd16-1536-11e7-9f34-3d7b76bd1569.png)

charlesbos commented:
That's odd. You should get the following:
![screenshot at 2017-03-30 10-18-15](https://cloud.githubusercontent.com/assets/9774166/24497128/523ea59e-1532-11e7-813c-da561c81ac06.png)
I did rebase the repo to remove the other commits, are you certain you have master?

sc0w commented:
@lukefromdc @raveit65 it works for you?

raveit65 commented:
Yes, i copied the image to /usr/share/themes/TraditionalOk/gtk-3.0/img/ by hand and added the settings to mate-application.css.
Of course we need to add the new image to MakeFile otherwise it won't installed.

charlesbos commented:
I'm so sorry! I forgot to do that. For convenience sake, I was just symlinking TraditionalOk & TraditionalGreen in ${HOME}/.themes to test changes.

sc0w commented:
now it works

squashed and merged in https://github.com/mate-desktop/mate-themes/commit/31fde6135ccffef75d05b19d931fceb08abacc40

thanks

sc0w commented:
I think with this colors the look is better:

https://github.com/sc0w/mate-themes/commit/ce9f58386cce33b80192d3e8e07cde7bffb4afbd

up: no changes
middle: use same color than @notebook_active_tab_strip_gradient_a
down: use same color than @notebook_active_tab_strip_gradient_b

TOK

![bluepanel](https://cloud.githubusercontent.com/assets/7734191/24545799/ffe41f7c-1608-11e7-9b20-d93bd998e61d.png)

TG

![greenpanel](https://cloud.githubusercontent.com/assets/7734191/24545807/07b41b80-1609-11e7-9889-5470a82337db.png)


@charlesbos what do you think? do you like it?

charlesbos commented:
Thanks for merging. Yes, the colours look good! I like that idea.

sc0w commented:
done!

https://github.com/mate-desktop/mate-themes/commit/96f8b3f56e1fa161fb7258f269a1773882e3a97f

:+1:
